docs/components/Content/Web.stories.tsx

```diff
@@ -45,4 +45,5 @@ const BasicTemplate: ComponentStory<typeof Content> = args => (
 export const Basic = BasicTemplate.bind({});
 Basic.args = {
   spacing: "small",
+  type: "section",
```

Comment:
hmmmmmm, now that I think about it. We shouldn't set our default example to a section since we can assume that the code for that (under the code tab) is what's gonna be copy-pasted everywhere.

If we really want to showcase that, we should add another example. IMO, it showing up under controls is enough to tell people that they can do this.

<img width="740" alt="image" src="https://github.com/GetJobber/atlantis/assets/15986172/5e299793-30dc-487f-b959-5bd0a7f93b96">

```suggestion
```

Reply:
Yeah I was honestly wondering the same thing. I don't know that this really needs to be demo'd given that it's not a visible change.